brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2570:  Implemented logic to append style tag inside blocks html to resolve an issue loading block dynamic CSS in frame and optimize font loader logic.

### Description

- Introduce separate generic component for dynamic CSS generation for blocks.
- Cleanup edit.js and memoaise font loader separate component in separate file for each blocks.

### Screenshots

- https://share.getcloudapp.com/DOulmY0R
- https://share.getcloudapp.com/9ZuKD1zv

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any block.
- Add styling to it.
- Inspect it, and make sure the style tag is appended within the block HTML as shown in [here](https://share.getcloudapp.com/DOulmY0R)
- Check by making it template part or switching it into tablet/mobile devices.

### Checklist:
